packages/react-sdk/src/components/elements/rectangle/Display.tsx

```diff
@@ -40,9 +40,26 @@ const RectangleDisplay = ({
   ...shape
 }: RectangleElementProps) => {
   const { rx, strokeColor, strokeWidth, text } = getRenderProperties(shape);
+  const stickyNote = shape.stickyNote;
+
+  const shadowFilter = (
```

Comment:
```suggestion
  const shadowFilter = (
    <filter id="bottom-shadow" x="-50%" y="-50%" width="200%" height="200%">
      <feDropShadow
        dx="0"
        dy="6"
        stdDeviation="5"
        floodColor="black"
        floodOpacity="0.33"
      />
    </filter>
  );
```

I think it went too light - I like this better.